source/install/docker-local-machine.rst

```diff
@@ -68,6 +68,31 @@ Ubuntu
 3. When docker is done fetching the image, open ``http://localhost:8065/``
    in your browser.
 
+Fedora
+^^^^^^
+
+1. Follow the instructions at https://docs.docker.com/engine/installation/linux/fedora/ or use the summary below:
+
+   ..  code:: bash
+      
+        sudo dnf -y install dnf-plugins-core
+        sudo dnf config-manager \
+        --add-repo \
+        https://download.docker.com/linux/fedora/docker-ce.repo
+        sudo dnf install docker-ce docker-compose git # Accepting the new docker repository key
+        sudo usermod -aG docker <username>
+        sudo systemctl start docker
+        cd mattermost-docker
+        sudo docker-compose build
+ 
+2. Start the container: 
+
+   .. code:: bash
+      
+       sudo docker-compose up -d
+       
+3. When the startup is complete, visit ``https://localhost/`` in your browser.  
```

Comment:
Step 3 in the procedures is like this:

>When Docker is done fetching the image, open http://localhost:8065/ in your browser.

Can you make yours consistent? Also, is it localhost, or localhost:8065?

If you have time, maybe you can fix a few typos in the other procedures while you're at it... they all seem to have "docker" instead of "Docker".

Thanks!